### ISSUE TITLE
How do I get started?

### ISSUE THREAD

**Issue by sgtpenguin** [dispositiontools/craft-alt-text-generator#5]
I have installed a trial of the plugin and added an API key.  It's not obvious to me how to begin generating alt text and I don't see any instructions.  Thanks

**Comment by dispositiontools** [comment]
Hi @sgtpenguin ,

ah - yes, I should make that clearer in the docs. 
There are a couple of different ways you can get started. 

If you go to Utilities > Alt Text Generator Utility > Queue all images with no alt text >Queue images

that will queue all the images with no alt text and send them to Attext.ai for alt text generation. 

Or you can add an asset to the queue from the asset listing page by selecting the ones you want and then click on the element actions menu (the little cog) and then select "Generate Alt Text"

or set the option in settings of 
Send to AltText.ai for Alt Text Generations when saving with no alt text 
to true, 

then when you upload a new asset it will generate the alt text automatically. 

Hope that helps!

**Comment by sgtpenguin** [comment]
Ah that makes a lot of sense.  Thank you!  Looking forward to trying it out
